icons/circle-equal.svg

```diff
@@ -0,0 +1,15 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <line x1="8" x2="16" y1="9" y2="9" />
+  <line x1="8" x2="16" y1="15" y2="15" />
```

Comment:
I think this has to be about the same ratio as our regular `equals` icon, so maybe something more like this?
```suggestion
<path d="M7 10H17" />
<path d="M7 14H17" />
```